flow/table.go

```diff
@@ -170,3 +170,31 @@ func (ft *FlowTable) FilterLast(last time.Duration) []*Flow {
 	ft.lock.RUnlock()
 	return flows
 }
+
+func (ft *FlowTable) SelectLayer(endpointType FlowEndpointType, list []string) []*Flow {
+	meth := make(map[string][]*Flow)
```

Comment:
Yeah, I forgot :( I updated the PR, we need a golang static analysis better than go-lint